tools/translate_core_files.sh

```
@@ -0,0 +1,344 @@
+- content/post/_index.md → content/post/_index.<lang_code>.md (posts section)
+- content/page/about/index.md → content/page/about/index.<lang_code>.md (if exists)
+- content/page/colophon/index.md → content/page/colophon/index.<lang_code>.md (if exists)
+- content/page/translations/index.md → content/page/translations/index.<lang_code>.md (if exists)
+
+EOF
+}
+
+# Function to get language code from language name
+get_language_code() {
+    local language="$1"
+    local lower_language=$(echo "$language" | tr '[:upper:]' '[:lower:]')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)